perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -140,6 +140,9 @@
                      'Defaults to None which uses the ycsb default of 10.')
 flags.DEFINE_integer('ycsb_field_length', None, 'Size of each field. Defaults '
                      'to None which uses the ycsb default of 100.')
+flags.DEFINE_enum('ycsb_requestdistribution',
```

Comment:
Please add comment that this will overwrite the requestdistribution parameter in workload file.

Reply:
added more info on this paramater.